sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeHealthcareEntitiesTests.cs

```diff
@@ -54,6 +54,8 @@ public AnalyzeHealthcareEntitiesTests(bool isAsync, TextAnalyticsClientOptions.S
         };
 
         [RecordedTest]
+        // Temporarily setting max version to V2022-05-01 since V2022-10-01-preview does not support AAD yet.
```

Comment:
Tip: use `// TODO: ...` because Visual Studio has a Tasks view that tracks these, and it's much easier to find all `TODO` comments this way rather than a random comment.

I also recommend, as we normally do, putting a link to the tracking bug so that you can also find all TODOs specific to a particular issue and fix them all.